tools/manage_clvm.py

```diff
@@ -212,25 +229,41 @@ def check(use_cache: bool) -> int:
     cache_modified = False
 
     found_stems = find_stems(top_levels)
-    for name in ["hex", "hash"]:
-        found = found_stems[name]
-        suffix = all_suffixes[name]
-        extra = found - found_stems["clvm"]
+    found = found_stems["hex"]
+    suffix = all_suffixes["hex"]
+    extra = found - found_stems["clsp"]
 
-        print()
-        print(f"Extra {suffix} files:")
+    print()
+    print(f"Extra {suffix} files:")
 
-        if len(extra) == 0:
-            print("    -")
-        else:
-            overall_fail = True
-            for stem in extra:
-                print(f"    {stem.with_name(stem.name + suffix)}")
+    if len(extra) == 0:
+        print("    -")
+    else:
+        overall_fail = True
+        for stem in extra:
+            print(f"    {stem.with_name(stem.name + suffix)}")
 
     print()
-    print("Checking that all existing .clvm files compile to .clvm.hex that match existing caches:")
+    print("Checking that no .clvm files begin with `(mod`")
     for stem_path in sorted(found_stems["clvm"]):
-        clvm_path = stem_path.with_name(stem_path.name + clvm_suffix)
+        with open(stem_path.with_name(stem_path.name + clvm_suffix)) as file:
+            file_lines = file.readlines()
+            for line in file_lines:
+                non_comment: str = line.split(";")[0]
+                if "(" in non_comment:
+                    paren_index: int = non_comment.find("(")
+                    if len(non_comment) >= paren_index + 4 and non_comment[paren_index : paren_index + 4] == "(mod":
+                        overall_fail = True
+                        print(f"FAIL    : {stem_path.name + clvm_suffix} contains `(mod`")
+                    break
+
+    missing_files: typing.List[str] = []
+    all_hash_stems: typing.List[str] = []
+
+    print()
+    print("Checking that all existing .clsp files compile to .clsp.hex that match existing caches:")
+    for stem_path in sorted(found_stems["clsp"]):
+        clvm_path = stem_path.with_name(stem_path.name + clsp_suffix)
```

Comment:
And just in general review all references to `clvm` since I think they should all be gone what with all the `.clvm` files being gone?

```suggestion
        clsp_path = stem_path.with_name(stem_path.name + clsp_suffix)
```

Reply:
https://github.com/Chia-Network/chia-blockchain/pull/14996#discussion_r1163237917